cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -255,10 +258,84 @@ func (m *awsWrapper) getInstanceTypeByLaunchTemplate(launchTemplate *launchTempl
 	return aws.StringValue(instanceType), nil
 }
 
+func (m *awsWrapper) getInstanceTypesFromInstanceRequirements(policy *mixedInstancesPolicy) ([]string, error) {
```

Comment:
This needs to handle regular LT Instance Requirements (ie. no overrides are present in the Mixed Instances Policy) as well.